### ISSUE TITLE
Install error because of the scipy version

### ISSUE THREAD

**Issue by mousset** [astro-informatics/s2scat#25]
Error when doing _import s2scat_ because of the scipy version 1.14.1. This is the error I got

---> 27 from scipy.integrate import trapz
ImportError: cannot import name 'trapz' from 'scipy.integrate'

By installing scipy==1.12.0, the issue was solved. Should we put this in the requirements ?

**Comment by CosmoMatt** [comment]
@mousset good catch, we've pinned some versions already but scipy must be picked up externally. If pinning scipy==1.12.0 fixes the problem for now let's do it. We can worry about a more comprehensive fix for versions across the board at a later date.

If you can make a small PR with this change, I can review quickly and we can merge this into main!
